tpu-client-next/tests/connection_workers_scheduler_test.rs

```diff
@@ -16,11 +16,13 @@ use {
         streamer::StakedNodes,
     },
     solana_tpu_client_next::{
-        connection_workers_scheduler::{ConnectionWorkersSchedulerConfig, Fanout},
+        connection_workers_scheduler::{
```

Comment:
nit: I don't think this file needs the _test suffix, I'd call it connection_workers_scheduler.rs

Reply:
I will postpone this for the later, because this change is not necessary for backporting